app/api/search/metadataAggregations.js

```diff
@@ -226,3 +226,74 @@ export const permissionsLevelAgreggations = baseQuery => {
     },
   };
 };
+
+export const publishingStatusAgreggations = baseQuery => {
+  const path = 'published';
+  const filters = extractFilters(baseQuery, path);
+  const { should } = baseQuery.query.bool;
+  const user = permissionsContext.getUserInContext();
+  const isCollab = user && !['admin', 'editor'].includes(user.role);
```

Comment:
Since it's more likely that a new role is permissions bounded, rather than super-user, this way it's already contemplated. So far, this is the way we have implemented this type of condition, so its also more consisten.